decidim-elections/decidim-elections.gemspec

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push File.expand_path("lib", __dir__)
+
+require "decidim/elections/version"
+
+Gem::Specification.new do |s|
+  s.version = Decidim::Elections.version
+  s.authors = ["Ivan Vergés"]
+  s.email = ["ivan@pokecode.net"]
+  s.license = "AGPL-3.0-or-later"
+  s.homepage = "https://decidim.org"
+  s.metadata = {
+    "bug_tracker_uri" => "https://github.com/decidim/decidim/issues",
+    "documentation_uri" => "https://docs.decidim.org/",
+    "funding_uri" => "https://opencollective.com/decidim",
+    "homepage_uri" => "https://decidim.org",
+    "source_code_uri" => "https://github.com/decidim/decidim"
+  }
+  s.required_ruby_version = "~> 3.3.0"
+
+  s.name = "decidim-elections"
+  s.summary = "A Decidim module for creating Elections."
+  s.description = "A module that aims to provide a set of tools to create elections in Decidim."
+
+  s.files = Dir["{app,config,db,lib,vendor}/**/*", "Rakefile", "README.md"]
```

Comment:
Check otu the pattern on other modules because we do a `git ls-files` here (we should probably update the generator if it isn't htere)

Reply:
Applied! Also, I've done to the collaborative-texts module as well